unittest/factory/diff_action.hpp

```diff
@@ -1,7 +1,8 @@
 ///////////////////////////////////////////////////////////////////////////////
 // BSD 3-Clause License
 //
-// Copyright (C) 2018-2020, University of Edinburgh
+// Copyright (C) 2018-2020, LAAS-CNRS
```

Comment:
Why did you change this?

Reply:
Oops my bad, I did a copy paste and I didn't notice `University of Edinburgh`. Sorry about that, I will change that.